repair_order_group/readme/DESCRIPTION.md

```
@@ -0,0 +1,5 @@
+This module allows to group several repair orders, managing them more convenient way. Such orders are synchronised in the following actions:
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)